esphome/components/mcp4461/output/mcp4461_output.cpp

```diff
@@ -0,0 +1,87 @@
+#include "mcp4461_output.h"
+#include <cmath>
+
+#include "esphome/core/helpers.h"
+#include "esphome/core/log.h"
+
+namespace esphome {
+namespace mcp4461 {
+
+static const char *const TAG = "mcp4461";
+
+void Mcp4461Wiper::write_state(float state) {
+  ESP_LOGV(TAG, "Got value %02f from frontend", state);
+  const float max_taps = 256.0;
+  state = state * 1000.0;
+  if (state > max_taps) {
+    ESP_LOGW(TAG, "Cannot set taps > 0.256 for wiper %d, clamping to 0.256!", this->wiper_);
```

Comment:
```suggestion
    ESP_LOGW(TAG, "Cannot set taps > 0.256 for wiper %" PRIu8 ", clamping to 0.256!", static_cast<uint8_t>(this->wiper_));
```

Or use somekind of constexpr tounderlying